javascript/ql/src/Security/CWE-020/IncompleteHostnameRegExp.ql

```diff
@@ -0,0 +1,64 @@
+/**
+ * @name Incomplete regular expression for hostnames
+ * @description Matching a URL or hostname against a regular expression that contains an unescaped dot as part of the hostname might match more than expected.
```

Comment:
Not sure I understand the last part ("might match more than expected") but feel free to ignore if you think users/developers will get it